alpha/apps/kaltura/lib/kPlaybackContextDataHelper.php

```diff
@@ -397,6 +397,18 @@ private function constructRemotePlaybackSources(entry $dbEntry, kContextDataHelp
 		if (!count($this->remoteFlavorsByDc))
 			return;
 
+		$partner = $dbEntry->getPartner();
+		if ($partner->getEnforceDelivery())
+		{
+			$deliveryAttributes = DeliveryProfileDynamicAttributes::init(null, $dbEntry->getId(), null);
```

Comment:
if this line is called anyway, they just move its call before this block and remove the call in line 412